SmartDeviceLink/SDLLockScreenPresenter.m

```diff
@@ -98,10 +99,11 @@ - (void)sdl_presentIOS13 {
             }
         }
 
+		// Find the currently visible app window
         NSArray<UIWindow *> *windows = appWindowScene.windows;
         UIWindow *appWindow = nil;
         for (UIWindow *window in windows) {
-            if (window != self.lockWindow) {
+            if (window.isKeyWindow) {
```

Comment:
I don't think this will work for apps using iOS 13 scene API. `keyWindow` is deprecated, though this API appears not to be. In any case, I wouldn't trust it.

Reply:
This code was removed